polkapox/1.0.0-beta/Dockerfile

```diff
@@ -0,0 +1,85 @@
+##### Building stage #####
+
+# Stage 1: Build Python environment with required packages
+FROM python:3.8-slim AS app
```

Comment:
This is correct, but we have a GA that checks for `as app` instead of `AS app`. Sorry! Could you change this?

Reply:
I made this change!